base/deprecated.jl

```diff
@@ -2093,6 +2093,11 @@ end
     @deprecate chol!(x::Number, uplo) chol(x) false
 end
 
+@deprecate_moved Nullable "Nullables"
+@deprecate_moved NullException "Nullables"
+@deprecate_moved isnull "Nullables"
+@deprecate_moved unsafe_get "Nullables"
```

Comment:
Same here. Was `unsafe_get` only used for `Nullable`?

Reply:
Yes.